src/components/report/Summary.js

```diff
@@ -9,7 +9,9 @@ export default {
 	},
 	computed: {
 		featureLength() {
-			const count = Object.keys(this.features).length;
+			const count =
+				Object.keys(this.features.fixed).length +
```

Comment:
I see in quite a few places the fixed and optional features are grouped together. I wonder if it wouldn't make more sense if the data looked something like this?

```
[
  {
     name: "ccmp",
     state: "fixed",
    ...
  },
  {
     name: "Case-Sensitive Forms",
     state: "optional"
  },
]
```

That makes more sense IMO if in the majority of cause you don't need to distinguish between fixed and optional. You can still create a helper like this, if you do need to distinguish:

```js
const getOptionalFeatures = features.filter(f => f.state == "optional");
```

Arrays are much easier to work with than objects.

Reply:
Great, I now refactored all of the feature stuff and indeed — you are right 😄 

Thanks! Please re-review :-)